action.yml

```diff
@@ -92,16 +93,30 @@ runs:
         target: ${{ inputs.target }}
         outputs: type=docker,dest=${{ runner.temp }}/local-docker-image.tar
 
+    - name: Setup python
+      uses: actions/setup-python@v5
+      with:
+        python-version: "3.x"
+
+    - name: Install TruffleHog
+      shell: bash
+      run: |
+        TRUFFLEHOG_VERSION="3.88.5"
+        echo "Installing TruffleHog version $TRUFFLEHOG_VERSION"
+        curl -sL "https://github.com/trufflesecurity/trufflehog/releases/download/v${TRUFFLEHOG_VERSION}/trufflehog_${TRUFFLEHOG_VERSION}_linux_amd64.tar.gz" -o trufflehog.tar.gz
+        tar -xzf trufflehog.tar.gz
+        chmod +x trufflehog
+
+        sudo mv trufflehog /usr/local/bin/
+
+        # Cleanup
+        rm trufflehog.tar.gz
+
     - name: Scan docker image with TruffleHog
+      id: scan-docker-image
       shell: bash
       run: |
-        docker pull trufflesecurity/trufflehog:latest
-        docker run --rm -v ${{ runner.temp }}:/tmp \
-          trufflesecurity/trufflehog:latest \
-          docker --image file:///tmp/local-docker-image.tar \
-          --github-actions \
-          --no-verification \
-          --fail
+        python $GITHUB_ACTION_PATH/docker-image-scanner.py ${{ runner.temp }}/local-docker-image.tar $GITHUB_ACTION_PATH/trufflehog-ignore
```

Comment:
Do we also need to support local trufflehog-ignore from the repos that use this action?
Maybe we can add support for it later if teams ask for it.